src/EditorFeatures/Text/Extensions.SnapshotSourceText.cs

```diff
@@ -35,14 +35,14 @@ private class SnapshotSourceText : SourceText
             private readonly Encoding? _encoding;
             private readonly TextBufferContainer? _container;
 
-            private SnapshotSourceText(ITextBufferCloneService? textBufferCloneService, ITextSnapshot editorSnapshot, SourceHashAlgorithm checksumAlgorithm, TextBufferContainer container)
+            private SnapshotSourceText(ITextBufferCloneService? textBufferCloneService, ITextSnapshot editorSnapshot, Encoding? encoding, SourceHashAlgorithm checksumAlgorithm, TextBufferContainer container)
```

Comment:
i'm missing the callsite that passed in a non-null value here.  or was this just to make it explicit that all callsites are trying to pass that in?

Reply:
The ChangedSourceText.ctor was previously calling the ITextImage version of this ctor, and passing in an encoding. I wanted to match that behavior.